pkg/controller/mizar/mizar-pod-controller.go

```diff
@@ -121,13 +121,18 @@ func (c *MizarPodController) updateObj(old, cur interface{}) {
 	}
 
 	key, _ := controller.KeyFunc(curObj)
-	c.queue.Add(KeyWithEventType{Key: key, EventType: EventType_Update})
+	c.queue.Add(KeyWithEventType{Key: key, EventType: EventType_Update, ResourceVersion: curObj.ResourceVersion})
 }
 
 func (c *MizarPodController) deleteObj(obj interface{}) {
 	key, _ := controller.KeyFunc(obj)
 	klog.Infof("%v deleted. key %s.", controllerForMizarPod, key)
-	c.queue.Add(KeyWithEventType{Key: key, EventType: EventType_Delete})
```

Comment:
Do we need a resourceVersion here as there is only one delete event for each pod? 

If that is the case, we can just put an empty string as the ResourceVersion.

Reply:
Yes, I think empty string is better. Updated.